app/controllers/users_controller.rb

```diff
@@ -0,0 +1,83 @@
+class UsersController < ApplicationController
+  before_action :find_indv_user, only: [:show, :edit, :update, :destroy]
+
+  def loginform
+    @user = User.new
+  end
+
+  #   def login
+  #     username = params[:user][:username]
+  #     user = User.find_by(username: username)
+  #     user = User.create(username: username) if user.nil?
+
+  #     if user.id
+  #       session[:user_id] = user.id
+  #       flash[:alert] = "#{user.username} logged in"
+  #       redirect_to root_path
+  #     else
+  #       flash[:error] = "Unable to log in"
+  #       redirect_to root_path
+  #     end
+  #   end
+
+  #   def current
+  #     @user = User.find_by(id: session[:user_id])
+  #     if @user.nil?
+  #       flash[:error] = "You must be logged in first!"
+  #       redirect_to root_path
+  #     end
+  #   end
+
+  #   def logout
+  #     user = User.find_by(id: session[:user_id])
+  #     session[:user_id] = nil
+  #     flash[:notice] = "Logged out #{user.username}"
+  #     redirect_to root_path
+  #   end
+
+  def index
+    @users = User.all
+  end
+
+  #   def new
+  #     @user = User.new
+  #   end
+
+  def create
+    new_user = User.new[:user_params]
```

Comment:
Instead of `User.new[:user_params`, you'll want to use the private method named `user_params` as an argument for the method `new` (which means `()`, not `[]`), so `new_user = User.new(user_params)`